app/models/ranking_entry.rb

```diff
@@ -0,0 +1,69 @@
+class RankingEntry
+  attr_reader :rank, :name, :match_count, :won_count, :draw_count, :lost_count, :goals, :goals_against, :goals_difference, :points
+  attr_accessor :rank
```

Comment:
@reviewers does anybody know what's the nicest way to deal with accessible attributes in ruby? Code climate gives a warning, might be better to create a accessor method? Or is this done internally?

Reply:
I'm pretty sure this is exactly what `attr_reader` and `attr_accessor` do. Not sure why codeclimates gives a warning tbh.